spec/requests/unlinking/unlink_defendant_maat_reference_v2_spec.rb

```diff
@@ -0,0 +1,297 @@
+# frozen_string_literal: true
+
+# rubocop:disable RSpec/MultipleMemoizedHelpers
+
+RSpec.shared_examples 'invalid unlink_attempt request for CD API' do
+  before do
+    patch "/defendants/#{defendant_id}?urn=#{prosecution_case_reference_from_fixture}",
+          params:
+  end
+
+  it 'does NOT send an unlink request to CD API' do
+    expect(a_request(:patch, api_request_path)
+          .with(body: api_request_payload.to_json, headers: json_content))
+      .not_to have_been_made
+  end
+
+  it 'renders edit' do
+    expect(response).to render_template(:edit)
+  end
+
+  it 'displays error summary with other_reason_text presence error' do
+    expect(response.body).to have_tag(:div, with: { class: 'govuk-error-summary' }) do
+      with_text error_message
+    end
+  end
+end
+
+RSpec.describe 'unlink defendant maat reference', type: :request, vcr: true, stub_unlink_v2: true do
```

Comment:
sorry to be pedantic, could you remove `vcr: true` because it's not used in this test.